### PR TITLE
Remove align attribute for HTML5 compability

### PR DESCRIPTION
**Pull request by Morendil**
As suggested in #30

### CODE REVIEW

File: lib/jemoji.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def emojify(doc)
       def filter_with_emoji(src)
         filters[src] ||= HTML::Pipeline.new([
           HTML::Pipeline::EmojiFilter
-        ], { :asset_root => src })
+        ], { :asset_root => src, img_attrs: Hash(align: nil) })
```
Comment (benbalter) [inline lib/jemoji.rb]: > `Hash(align: nil)`

Should be `{ align: nil }`

Comment (Morendil) [inline lib/jemoji.rb]: Thanks; done!

**Comment by benbalter** [comment]
@jekyllbot: merge +minor.

**Comment by benbalter** [comment]
Thanks @Morendil! ✌️ 🌵 🍔

**Pull request merged**